07 - Blocks/coding_exercise_block_variables.rb

```diff
@@ -0,0 +1,50 @@
+# Define an increments_of_two method that accepts no arguments. 
+# It should output the multiples of 2 from 0 to 10. 
+# Output the multiples in order with no spaces between them.
+# Use the print method in your solution.
+#
+# Example:
+# increments_of_two() should output "0246810"
+
+def increments_of_two
+  6.times do |count|
+
+    print count*2
+    
+  end
+end
+increments_of_two
+puts
+puts
+=begin
+Exercise 1:
```

Comment:
it would be clearer if the comments and code were structured together for example